iceoryx_utils/test/moduletests/test_posix_timer.cpp

```diff
@@ -406,3 +407,117 @@ TEST_F(Timer_test, GetOverrunsFailsWithNoCallback)
     ASSERT_THAT(call.has_error(), Eq(true));
     EXPECT_THAT(call.get_error(), Eq(TimerError::TIMER_NOT_INITIALIZED));
 }
+
+TIMING_TEST_F(Timer_test, SoftTimerContinuesWhenCallbackIsLongerThenTriggerTime, Repeat(5), [&] {
```

Comment:
please adjust name, this is now `CatchUpPolicySkipToNextBeat` instead of `SoftTimer`